integration-libs/cdc/manage-users/manage-users.service.spec.ts

```diff
@@ -0,0 +1,117 @@
+import { TestBed } from '@angular/core/testing';
```

Comment:
Move this spec file to `manage-users-list`

Reply:
done